### ISSUE TITLE
Meteor 2.5.6 Windows Build Process: TARBALL Permissions Problem

### ISSUE THREAD

**Issue by sotarules** [meteor/meteor#11877]
### This bug report should include:
Meteor 2.5.6 Windows Build Process: TARBALL Permissions Problem
Error occurs on Meteor 2.5.6
Meteor 2.5.3 Works Correctly
Building on Windows 10 for Ubuntu 20.04
meteor build \bundles --architecture os.linux.x86_64
When running npm install --unsafe-perm we get many permissions errors.  Excerpt:
```
tar: bundle/programs/server/npm/node_modules/lodash/sortedIndexOf.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedLastIndex.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedLastIndexBy.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedLastIndexOf.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedUniq.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedUniqBy.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedIndexOf.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedLastIndex.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedLastIndexBy.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedLastIndexOf.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedUniq.js: Cannot open: Permission denied
tar: bundle/programs/server/npm/node_modules/lodash/sortedUniqBy.js: Cannot open: Permission denied
```
To reproduce simply install Meteor 2.5.6 on Windows, then run a build for os.linux.x86_64 architecture.
Transfer TAR to target Linux system (Ubuntu 20.04 in my case)
```
cp ~/Downloads/keeptrack.tar.gz ~/bundles/keeptrack/
cd ~/bundles/keeptrack
tar -xzf keeptrack.tar.gz
chmod 755 -R ~/bundles/keeptrack
cd ~/bundles/keeptrack/bundle/programs/server
npm install --unsafe-perm
```
By the way, if we run the tar under sudo, it seems to work:
```
cd ~/bundles/keeptrack
sudo tar -xzf keeptrack.tar.gz
sudo chmod 755 -R ~/bundles/keeptrack
sudo chown -R ubuntu:ubuntu ~/bundles/keeptrack
cd ~/bundles/keeptrack/bundle/programs/server
npm install --unsafe-perm
```
You can contact me anytime at sotarules@gmail.com for more information. Thanks in advance for all you do, we love Meteor.

This makes sense if the permissions in the tar have some how been impacted between 2.5.3 and 2.5.6 - my hunch this happens on Windows only

**Comment by renanccastro** [comment]
hi @sotarules 
Thanks for reporting, there is indeed a change on 2.5.3 to 2.5.6 in the way of generating/dealing with tarballs.

I've fixed the flow of extracting the tarball inside meteor-tool, but outside of it wasn't covered. The "mode" header when creating the tar need to be changed when creating a tar on windows.

Please follow this PR https://github.com/meteor/meteor/pull/11879. Which should follow in a upcoming release.

I assume this issue has a workaround for no, right? Let me know if that's not the case.

**Comment by sotarules** [comment]
Yes work around is to untar as sudo then update ownership of resulting files.

**Comment by denihs** [comment]
Hi @sotarules, the fix will land on 2.6.
